### ISSUE TITLE
Handle 'sponsored ads' on Instagram web page

### ISSUE THREAD

**Issue by dhowe** [dhowe/AdNauseam#2370]
https://www.instagram.com/

These newly appearing ads generally have external 'shop now' or 'learn more' links at bottom:

<img width="556" alt="image" src="https://github.com/dhowe/AdNauseam/assets/737638/8bfac88d-5555-4f44-b928-29f5eccfd6f3">

<img width="522" alt="image" src="https://github.com/dhowe/AdNauseam/assets/737638/a6539a92-f1c5-4907-b20f-aecf39cd7591">

**Comment by mneunomne** [comment]
1. I couldn't reproduce this issue on my side, and I imagine this issue is actually because of not updated filter list (@dhowe can you test updating the filter lists?). Seems that also uBlock version beyond our current one has a smaller update period (from 4hours to 2hours) https://github.com/gorhill/uBlock/commit/885b3ea6c0600b1e9d50fc1100b5cc6a3724d166. Once pulled this perhaps will improve on our side as well.

2. Instagram, like facebook, is not really possible to collect their ads for the way html is structured and constantly changing. So first of all, maybe its a good idea to add instagram on the default strict-blocked webpages.

**Comment by dhowe** [comment]
1. Note that you need to be logged in (geo=US)
2. Yes, to default strict-block

**Comment by mneunomne** [comment]
Ok, added instagram to default strict-blocked.

**Comment by dhowe** [comment]
Will this also catch 'suggested for you' posts?

**Comment by mneunomne** [comment]
No, suggested for you is just instagram trying to be tiktok and just showing us global content... Now facebook also has the same thing.

**Comment by dhowe** [comment]
Still seeing sponsored ads on Instagram, does this need to wait until next release?

<img width="512" alt="image" src="https://github.com/dhowe/AdNauseam/assets/737638/9d3d40a9-0286-44d8-b885-e6bbd285332d">

**Comment by mneunomne** [comment]
Ads on instagram are currently being successfully blocked. Same for you @dhowe ?

**Comment by dhowe** [comment]
Seems so, I'll re-open if this changes

**Comment by dhowe** [comment]
<img width="552" alt="Screenshot 2023-10-15 at 12 53 08 AM" src="https://github.com/dhowe/AdNauseam/assets/737638/6db4c4a8-d57f-4918-8403-8242fae8b400">

**Comment by dhowe** [comment]
Still appearing in 'strict' mode:

![image](https://github.com/dhowe/AdNauseam/assets/737638/fbe08696-8b63-4e30-b3a8-8a6cd3eb62df)

![image](https://github.com/dhowe/AdNauseam/assets/737638/70c3bd7e-0425-4eaf-93de-d259914061c4)
